### DISCUSSION TITLE
Unable to produce the report.pdf

### DISCUSSION THREAD

**Discussion by akhilmanga** [Cyfrin/security-and-auditing-full-course-s23#8] [Q&A] [answered]
- I'm unable to produce the report.pdf

- I asked AI too, but it did not help

- when i do '' pandoc report.md -o report.pdf --from markdown --template=eisvogel --listings '' -> it is showing the error

<img width="490" alt="Screenshot 2023-12-07 at 8 51 10 AM" src="https://github.com/Cyfrin/security-and-auditing-full-course-s23/assets/116151859/ead1464b-f11f-4eed-b66a-db37bf056be8">

**Comment by cromewar** [comment]
@PatrickAlphaC  a clue on this one please?

**Reply by akhilmanga** [reply, marked as answer]
Just remove \* from \vspace\* to find it in your report.md file. After that run the command => '' pandoc report.md -o report.pdf --from markdown --template=eisvogel --listings ''. You will be able to see report.pdf

**Reply by denissosnowsky** [reply]
Thanks, it helped me!

**Reply by cromewar** [reply]
thanks for the answer @akhilmanga

**Reply by GalBarZakay** [reply]
HI! Hope to resolve this issue with this solution.. Where should I remove * from \vspace* from ?

**Comment by TilakMaddy** [comment]
For others,  https://md2pdf.netlify.app/ also seems to do a pretty good job

**Comment by 0xbl4ckwlf** [comment]
I also keep getting this same error after installing pandoc and texlive
How do I go about it?
PS: I use Windows 
```
Error producing PDF.
! LaTeX Error: Missing \begin{document}.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.1 {"
```
